guides/micronaut-data-access-mybatis/micronaut-data-access-mybatis.adoc

```diff
@@ -0,0 +1,207 @@
+= @guideTitle@
+
+@guideIntro@
+
+Authors: @authors@
+
+Micronaut Version: @micronaut@
+
+== Getting Started
+
+Learn how to access a database with MyBatis using Micronaut.
+
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-create-app.adoc[]
+
+=== Configure Data Source and JPA
+
+Add the next snippet to `build.gradle` to include the necessary dependencies:
+
+:dependencies:
+
+dependency:mybatis:3.4.6[groupId=org.mybatis,callout=1]
```

Comment:
Please make sure you use the latest MyBatis version: https://mvnrepository.com/artifact/org.mybatis/mybatis